scripts/export_tasks.py

```diff
@@ -0,0 +1,40 @@
+import ast
+import os
+import pathlib
+import json
+
+lib_to_task_map = {}
+
+
+def extract_tasks(library_name, variable_name, value):
+    if variable_name == "ALLOWED_TASKS":
+        if isinstance(value, ast.Dict):
+            for key in value.keys:
+                lib_to_task_map.setdefault(library_name, []).append(key.value)
+
+
+def traverse_global_assignments(library_name, file_content, handler):
+    for element in ast.parse(file_content).body:
+        # Typical case
+        if isinstance(element, ast.AnnAssign):
+            handler(library_name, element.target.id, element.value)
```

Comment:
Out of curiosity, what is `target.id` here?

Reply:
Sure. target is name. it contains both id which is the variable name and also a context to differentiate loading, deleting and assignment. Ref: https://docs.python.org/3/library/ast.html#ast.Store